build-system/tasks/performance/measure-documents.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const puppeteer = require('puppeteer');
```

Comment:
I think `puppeteer` will have to be lazy-required like the visual-diff tests do:

https://github.com/ampproject/amphtml/blob/bbd96aab653a682a8d2426356cb45b884feb7e13/build-system/tasks/visual-diff/index.js#L761-L771